app/components/explore/side-bar.js

```diff
@@ -150,6 +151,10 @@ export default Component.extend({
 
     toggleFilters() {
       this.set('showFilters', !this.showFilters);
+    },
+
+    hideMap() {
```

Comment:
The action is called hideMap. But what is doing is actually toggling the property and not hiding it. So you should probably call this action as `toggleMap`

Reply:
Done